client/src/components/menubar/style.ts

```diff
@@ -5,14 +5,8 @@ import { FEATURES } from "../../util/featureFlags/features";
 
 export const MAX_VERTICAL_THRESHOLD_WIDTH_PX = 500;
 const isTest = getFeatureFlag(FEATURES.TEST);
-const isDownload = getFeatureFlag(FEATURES.DOWNLOAD);
 
-let FIRST_VERTICAL_THRESHOLD_WIDTH_PX = 655;
-if (isTest && isDownload) {
-  FIRST_VERTICAL_THRESHOLD_WIDTH_PX = 705;
-} else if (isTest || isDownload) {
-  FIRST_VERTICAL_THRESHOLD_WIDTH_PX = 685;
-}
+const FIRST_VERTICAL_THRESHOLD_WIDTH_PX = isTest ? 705 : 685;
```

Comment:
@rainandbare PTAL if my math is correct here lol I'm thinking that now we have removed `isDownload` feature flag,  it should mean that the default width has increased from 655 to 685. And then if `isTest` is `true`, we bump it up to 705?

Thank you!

Reply:
If there is only ever one potential variable button then we can do: 

```suggestion
const FIRST_VERTICAL_THRESHOLD_WIDTH_PX = isTest ? 685 : 655;
```

Reply:
Chatting with Suzette about this in Slack at the moment! Will share the result here later 👍

Reply:
Confirmed that the current implementation is correct! Thanks again for your help, Suzette 🎉 !